aikiblog/templates/add_techniques.html

```diff
@@ -50,7 +50,7 @@
 			<td>{{ technique.date|date:" d.m.Y " }}&nbsp;&nbsp;</td>
 			<td>{{ technique.stand }}&nbsp;&nbsp;</td>
 			<td>{{ technique.attack }}&nbsp;&nbsp;</td>
-			<td>{{ technique.technique }}&nbsp;&nbsp;</td>
+			<td><a href="{% url "technique-list" technique.technique.slug %}">{{ technique.technique }}</a>&nbsp;&nbsp;</td>
```

Comment:
`technique-list` powinno być `technique_list`, widzę, że w urlach zastosowałeś też myślnik. Nie ma spotkałam się z taką pythonową konwencją nazywania
`technique.technique`- a cóż to takie paskudne nieco? Czemu technika ma jeszcze technikę?